Gruntfile.js

```diff
@@ -7,31 +7,25 @@ module.exports = function(grunt) {
       options: {
         separator: ';',
       },
-      dist: {
+      // Components installed with bower
+      external_libs: {
         src: [
-
-          // External Libraries
           'bower_components/jquery/dist/jquery.js',
           'bower_components/underscore/underscore.js',
           'bower_components/moment/moment.js',
           'bower_components/d3/d3.js',
           'bower_components/semantic-ui/dist/semantic.js',
-          // 'bower_components/leaflet-d3/dist/leaflet-d3.js',
-
-          // External Libraries (stored in version control)
-          'lib/ui/jquery.nouislider.min.js',
-          'lib/ui/jquery.daterangepicker.js',
-          'lib/xml.min.js',
```

Comment:
Good idea to clear these out. These are a great example of "legacy" functionality that we're not using atm.